doc/ansible/dev/advanced_options.md

```diff
@@ -152,3 +155,17 @@ spec in our `deploy/operator.yaml` might look something like:
     - name: ANSIBLE_VERBOSITY_MONGODB_DB_EXAMPLE_COM
       value: "4"
 ```
+
+Once the Operator is deployed, the only way to change the verbosity is via the
+`"ansible.operator-sdk/verbosity"` annotation. Continuing with our example, our
+CR may look like:
+
+```yaml
+apiVersion: "db.example.com/v1"
+kind: "PostgreSQL"
+metadata:
+  name: "example-db"
+  annotations:
+    "ansible.operator-sdk/verbosity": 5
+spec: {}
+```
```

Comment:
I think we should keep one example for each form to add to change the verbosity. WDYT?

Reply:
I'm not certain I understand. There is:

1. Command line arguments example on line 152
1. Environment variable example on line 156
1. CR annotation example being added here

Are you asking for the examples to be more explicit (said another way, would you like for the different examples to be more clearly enumerated)?

Reply:
Should be added the annotation and the env var always? Or we can apply it in a way and another?
So, I understand that we have 2 ways to do it. Then, is required 2 examples. Is it make sense?

Reply:
Was not the example with env var removed? Is this logic removed as well?